### ISSUE TITLE
MIME types not registered on Windows

### ISSUE THREAD

**Issue by cpiker** [autoplot/installers#3]
It looks like the Autoplot installer needs a little more work.

I just finished searching the Windows registry for a computer with Autoplot installed and didn't find the string:  vnd.autoplot.pngwalk which I think is the expected MIME type for pngwalk files.

Looking for just the value "pngwalk", I did find PNGwalk_.pngwalk under the key `ApplicationAssociatedToasts` and .pngwalk associated with Autoplot.exe the `FileExts` key.

Bottom line:  I think the installer only sets the file extension and not the mime type.

Can anyone verify that the expected MIME type for Autoplot PNGwalk files is:
```
text/vnd.autoplot.pngwalk
```
?

**Comment by jbfaden** [comment]
See also https://sourceforge.net/p/autoplot/feature-requests/744/

**Comment by cpiker** [comment]
Using MS Edge it seems the .vap file is associated with Autoplot, but not .pngwalk if that's a clue to the error in the installer.  The image below is a screenshot of the popup I get after clicking on a .vap file link in the browser.  The Autoplot icon is associated with the file name.

![vap_registered](https://user-images.githubusercontent.com/7530814/121246897-3f0dfc80-c867-11eb-886f-05b85e21bd0d.png)

When I click on a .pngwalk, Edge just displays the text file.

**Comment by cpiker** [comment]
@VoyagerPWS, @jbfaden I think we have this fixed.  I just ran a fresh install of the current package hosted at https://das2.org/autoplot and forced a browser reload (to ignore the local cache) and Autoplot was associated with the .pngwalk file automatically.

Will need verification, but I think this issue is closed.  The error was with the test server (space) not the windows registry entries created by the installer.

**Comment by cpiker** [comment]
[PDS-PPI node]( https://pds-ppi.igpp.ucla.edu/) conforms that the bug is fixed, closing the issue.
